configuration/javascript/http/order-processor/index.js

```diff
@@ -0,0 +1,82 @@
+import axios from "axios";
+import express, { json } from "express";
+
+const DAPR_HOST = process.env.DAPR_HOST ?? "localhost";
+
+let DAPR_PORT = process.env.DAPR_HTTP_PORT ?? 3500;
+let APP_PORT = process.env.APP_PORT ?? 6001;
+
+const DAPR_CONFIGURATION_STORE = "configstore";
+const BASE_URL = `http://${DAPR_HOST}:${DAPR_PORT}/v1.0-alpha1/configuration/${DAPR_CONFIGURATION_STORE}`;
+const CONFIGURATION_ITEMS = ["orderId1", "orderId2"];
+
+const app = express();
+app.use(express.json());
+
+async function main() {
+  // Get config items from the config store
+  CONFIGURATION_ITEMS.forEach((item) => {
+    axios
+      .get(`${BASE_URL}?key=${item}`)
+      .then((response) => {
+        console.log("Configuration for " + item + ":", response.data);
+      })
+      .catch((error) => {
+        console.log("Could not get config item, err:" + error);
+        process.exit(1);
+      });
+  });
+
+  // Start server to receive config updates
+  readConfigurationChanges();
+  // Subscribe to config updates
+  var subscriptionId = await subscribeToConfigUpdates();
+
+  // Unsubscribe to config updates and exit app after 20 seconds
+  setTimeout(async () => {
+    await unsubscribeToConfigUpdates(subscriptionId);
+    process.exit(0);
+  }, 20000);
+}
+
+async function subscribeToConfigUpdates() {
+  // Add delay to allow app channel to be ready
+  await sleep(3000);
```

Comment:
Do we need sleeps? What exactly are we waiting for here? Dapr has health checks that we can use for this instead right?

Reply:
Dapr waits for the app to be ready on the app-port. However, if I try to subscribe to config updates before the app is ready, I'll get an error since the app channel was not ready. App health check seems to be a preview feature and will require passing additional parameters to the CLI command, adding complexity to quickstarts.